tcms/rpc/tests/test_auth.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=attribute-defined-outside-init
+
+from xmlrpc.client import Fault as XmlRPCFault, ProtocolError
+from tcms.rpc.tests.utils import APITestCase
+
+
+class TestAuthLogin(APITestCase):
+    """Test Auth.login method"""
+
+    def _fixture_setup(self):
+        super()._fixture_setup()
+
+        self.rpc_client.Auth.logout()
+
+    def test_login_with_username(self):
+        sesson_id = self.rpc_client.Auth.login(self.api_user.username, "api-testing")
+        self.assertIsNotNone(sesson_id)
+
+    def test_login_without_password(self):
+        with self.assertRaisesRegex(XmlRPCFault, "Username and password is required"):
+            self.rpc_client.Auth.login(self.api_user.username, "")
```

Comment:
This covers the if condition on line 33-34 in the method. However we need another scenario where username/password aren't empty but aren't correct either to cover the PermissionDenied exception on line 41.